cmake/install.cmake

```diff
@@ -11,7 +11,13 @@ if (protobuf_BUILD_PROTOC)
   list(APPEND _install_targets libprotoc)
 endif ()
 
-foreach(_library ${_install_targets})
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/protobuf.pc.cmake
```

Comment:
I allowed this change from upstream. Not sure why we didn't have pkg-config files. Even if we don't use them, not worth removing from the build.